sklearn/linear_model/sag.py

```diff
@@ -170,6 +170,8 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
     Ridge, SGDRegressor, ElasticNet, Lasso, SVR, and
     LogisticRegression, SGDClassifier, LinearSVC, Perceptron
     """
+    if not warm_start_mem:
+        warm_start_mem = dict()
```

Comment:
I don't think for a simple init there is that much difference.

2015-10-31 12:53 GMT+01:00 Joel Nothman <notifications@github.com>:

> In sklearn/linear_model/sag.py
> <https://github.com/scikit-learn/scikit-learn/pull/5648#discussion_r43571067>
> :
>
> > @@ -170,6 +170,8 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
> >      Ridge, SGDRegressor, ElasticNet, Lasso, SVR, and
> >      LogisticRegression, SGDClassifier, LinearSVC, Perceptron
> >      """
> > +    if not warm_start_mem:
> > +        warm_start_mem = dict()
>
> Surely use a literal?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5648/files#r43571067>.
>